Formula/shogun.rb

```diff
@@ -0,0 +1,115 @@
+class Shogun < Formula
+  desc "Large scale machine learning toolbox"
+  homepage "http://www.shogun-toolbox.org"
+  url "http://shogun-toolbox.org/archives/shogun/releases/6.1/sources/shogun-6.1.3.tar.bz2"
+  sha256 "57169dc8c05b216771c567b2ee2988f14488dd13f7d191ebc9d0703bead4c9e6"
+
+  bottle do
+    sha256 "06aafb46d4cd4e53ebed6f8bb32e1279b5a8b5de9aae2359261cee9de87fc2ca" => :sierra
+    sha256 "e5dd5b447ef40488fb32fa36e2eb943975a6b28db8ae27859e39cb8a87a077a9" => :el_capitan
+    sha256 "b3d320d345da09b55ffb98a59f4fac3b2f6c2672fcbcda693fe0e7a9ee4795a3" => :yosemite
+    sha256 "dfbd03cba5e1a134a520d6c06aceaa3c5143cad638fc208150d980a44e5252cf" => :x86_64_linux
+  end
+
+  depends_on "pkg-config" => :build
+  depends_on "cmake" => :build
+  depends_on "xz"
+  depends_on "lzo"
+  depends_on "snappy"
+  depends_on "hdf5"
+  depends_on "json-c"
+  depends_on "readline"
+  depends_on "nlopt"
+  depends_on "eigen"
+  depends_on "arpack"
+  depends_on "glpk"
+  depends_on "r" => :optional
+  depends_on "lua" => :optional
+  depends_on "octave" => :optional
+  depends_on "opencv" => :optional
+  depends_on :java => :optional
+  depends_on :python => :optional
+
+  if build.with? "python"
+    include Language::Python::Virtualenv
```

Comment:
please don't use the Virtualenv DSL. Instead use ` libexec/"vendor/lib/python2.7/site-packages"` as in the `aws-shell` formula